config.cson

```diff
@@ -6780,11 +6780,13 @@ fileIcons:
 			[".ztl", "auto-red", alias: "ZBrush tool"]
 		]
 
-	Zeit:
-		icon: "zeit"
+	Vercel:
+		icon: "vercel"
 		match: [
 			[/^now\.json$/i, "auto-grey", priority: 2, uses: "JSON"]
 			[".nowignore", "auto-maroon", uses: "Ignore List"]
+			[/^vercel\.json$/i, "auto-grey", priority: 2, uses: "JSON"]
+			[".vercelignore", "auto-maroon", uses: "Ignore List"]
```

Comment:
Might be better to merge these into the existing entries:

~~~cson
match: [
	[/^(vercel|now)\.json$/i, "auto-grey", priority: 2, uses: "JSON"]
	[/^\.(vercel|now)ignore$/i, "auto-maroon", uses: "Ignore List"]
]
~~~